tuned/admin/admin.py

```diff
@@ -337,10 +340,7 @@ def _action_verify_profile(self, ignore_missing):
 		return False
 
 	def _action_dbus_off(self):
-		# 25 seconds default DBus timeout + 5 secs safety margin
-		timeout = 25 + 5
-		self._log_token = self._controller.log_capture_start(
-				self._log_level, timeout)
+		self._log_capture_start()
```

Comment:
I think the check for `self._log_level` should be here as well as the other places. Can you please add it, in a separate commit?